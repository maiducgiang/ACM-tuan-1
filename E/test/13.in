100
204685906 425697321
748269471 87581516
834956635 736571664
489317971 866734574
619215670 999122738
920728235 744769549
894647589 779697335
984745199 916225482
543242993 682051186
533743572 213571908
302319563 24771172
878355207 573101080
255572336 624365289
126969173 724131433
329067531 711203348
127763143 295079113
210735900 893398576
452733516 590899667
434028589 204921976
22910046 590987462
385274369 754641046
816930535 352214903
637926810 113399182
78007468 552355729
653005377 133059859
896836649 778383437
312820875 482148248
53606744 273078205
731155513 213055502
987633968 728792966
620007381 323257155
748397562 830513265
113027112 156822240
260333480 444188847
968396075 680108544
727645830 443893979
883890634 899803664
447804666 384704562
770497744 516613956
159943292 399178980
995580009 320827073
100099632 4565704
449714384 595329444
997357146 802896385
314741020 974039189
683350812 801856748
514273258 207132380
505687229 429914504
656248330 871988706
494437099 87251598
129475206 336755569
8772988 314475553
207283516 373538235
29218879 568102092
898184319 366395583
548886623 82727694
486001662 97558307
787487877 276669889
248028205 360254004
757271405 933267872
473054323 261872897
779445616 87101397
916041353 138179726
419493609 552963554
917624494 927189042
833461525 722698288
591257225 79422266
520642421 816333551
288453904 247319050
618960430 486617970
99835461 398332562
359821397 167932497
762864163 967821755
550921999 992664738
132631945 419658965
425728502 365780352
277007512 355232857
166633473 908959612
462639856 665935397
802121177 144786380
603327959 600999628
74159785 994326488
262899616 79900315
21265710 720356244
762479773 866353852
789414331 18483565
909006031 296769175
542766043 197346526
872290387 985732752
356681901 630196559
267148671 53510025
283298523 867827886
232741125 59712123
40921629 81969712
352697358 515951044
228235190 784015613
582825740 654119125
901934907 461323498
228370485 933687617
726419736 201589723
