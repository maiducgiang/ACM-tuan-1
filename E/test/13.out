3
2
2
2
2
2
2
2
2
2
2
2
3
6
3
3
5
2
2
26
2
2
2
8
2
2
2
6
2
2
2
2
2
2
2
2
2
2
2
3
2
2
2
2
4
2
2
2
2
2
3
36
2
20
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
4
2
2
2
4
2
2
6
2
2
2
14
2
34
2
2
2
2
2
2
2
4
2
3
2
4
2
2
5
2
